src/test/java/org/mockitoutil/Stopwatch.java

```diff
@@ -0,0 +1,51 @@
+package org.mockitoutil;
+
+import static java.lang.System.nanoTime;
+import static java.util.concurrent.TimeUnit.NANOSECONDS;
+import static org.junit.Assert.fail;
+
+import java.util.concurrent.TimeUnit;
+
+public class Stopwatch {
+
+	private Long startNanos;
+	
+	private Stopwatch() {
+	}
+	
+	public static Stopwatch createNotStarted(){
+		return new Stopwatch();
+	}
+	
+	
+	public void start(){
+		if (startNanos!=null)
```

Comment:
I am really sorry to comment, but this file seems to be indented with 8 spaces. Also please put whitespace before `{` and `!=` and alike.

This really makes me wonder whether we should reenable Checkstyle, since we (the Mockito developers) are not doing a good job on stating how we would like our code to be formatted to external contributors. Commenting about formatting for me feels very negative towards these awesome external contributions.